react/features/filmstrip/subscriber.web.js

```diff
@@ -26,7 +26,12 @@ import './subscriber.any';
  * Listens for changes in the number of participants to calculate the dimensions of the tile view grid and the tiles.
  */
 StateListenerRegistry.register(
-    /* selector */ getParticipantCountWithFake,
+    /* selector */ state => {
+        return {
+            count: getParticipantCountWithFake(state),
+            disableSelfView: state['features/base/settings'].disableSelfView
+        };
+    },
     /* listener */ (numberOfParticipants, store) => {
```

Comment:
Also, doesn't the listener need to be updated? It will not get an object instead of just the number.